### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by maaloum**
In this project, I add :
- navbar to index.html
- Media queries to control different sizes.
- Linters
- Update style.css

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -372,3 +383,84 @@ textarea:focus {
...
+  .container > nav {
+    display: inline-block;
+    font-weight: 500px;
+    color: #344563;
+  }
+
+  .works > .job {
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
```
Comment (omarsalem7) [inline style.css:418]: - [ ] consider matching your design like the one in the Figma template by specifying the correct padding and margins for work cards section 

- [ ] Also consider adding the correct background color for the work section like the Figma design.
- [ ] Kindly add appropriate padding and margin for about section

| Your design  | Figma design|
| ------------- | ------------- |
| image| image |
| image | image
 |

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @maaloum 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi  @maaloum  ✋ , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

ALT

 ## Highlights
- Linters are passing ✔️ 
- NIce use of Github flow 🥇 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
## Hi @maaloum, 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥 


## Highlights✨
- No linters error ✅✅
- Great job on working on the requested changes ✅🔥🔥
- Good commit messages 🔥🔥

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**